packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -157,5 +182,7 @@ export class NodeSDK {
 
       metrics.setGlobalMeterProvider(meterProvider);
     }
+
+    new NodePluginManager(this._pluginManagerConfig);
```

Comment:
I know this was recently discussed for not returning the value but it feels off to new up `NodePluginManager` and not have any handle on it. Looks like you had issues getting the linting to ignore an unused variable but perhaps we can get that figured out.

Part of this awkwardness perhaps comes from the existing pattern of loading via constructor VS a `load()`/other additional call on the plugin manager. Its a minor thing but I'm a fan of leaving constructors responsible for construction and then having equivalent function pairs of concepts like start/stop, load/unload, etc. Perhaps that ends up awkward, with how this is retrofitting the prior TracerProvider(s), but a thought. I'm relatively new to the project and preferences of existing maintainers, so would lean on their opinions more than mine.

Reply:
It might be a good idea to split the loading functionality out into a `load` function. Maybe we should do that in a follow-up PR though?

Reply:
👍 Yeah, not looking to hold things up. Just seeing an object created and then immediately released into the world makes me 👀 . In this case, we'll have a plan for it. In other cases, may indicate should just have a function, etc.

Reply:
yeah I agree, it's definitely needs to be changed. I mentioned this problem in one of the threads above and I'm looking forward to know how everyone wants to resolve this.
But you're right, the reason for this is that the reference variable would be unused and the linter doesn't allow the use of `@ts-ignore`.
Perhaps we can create the sdk `unload` method for now and just call the plugin manager's `stop()` in there for now and complete it in a followup PR. I appreciate it if everyone can share their thoughts.

Reply:
if we cannot find any reasonable usage of plugin manager then we might think of making this part of something else and exposing some method. I haven't thought much yet about it, but maybe the `opentelemetry` itself can have a method to load the plugins. This method could also take a param to allow provide the the tracer and meter providers.